config/config.go

```diff
@@ -160,13 +161,17 @@ func ParseFromLatest(rawConfig []byte) (types.Config, report.Report, error) {
 	return config, r, nil
 }
 
-func ParseFromV1(rawConfig []byte) (types.Config, error) {
+func ParseFromV1(rawConfig []byte) (types.Config, report.Report, error) {
 	config, err := v1.Parse(rawConfig)
 	if err != nil {
-		return types.Config{}, err
+		return types.Config{}, report.Report{}, err
 	}
 
-	return TranslateFromV1(config), nil
+	convertedConfig := TranslateFromV1(config)
+
+	rpt := validate.ValidateWithoutSource(reflect.ValueOf(convertedConfig))
+
+	return convertedConfig, rpt, nil
```

Comment:
Needs to return ErrInvalid if rpt.IsFatal

Reply:
Done

Reply:
should return an empty config in this case.

Reply:
Done